felix/dataplane/linux/int_dataplane.go

```diff
@@ -1564,15 +1564,13 @@ func (d *InternalDataplane) setUpIptablesBPF() {
 			})
 		}
 
-		// Accept any SEEN packets that go to the host. What remains here should be from
-		// host interfaces. This should accept packets in default-DROP iptable
-		// environments.  Such packets go through BPF on host iface but must go through
-		// the INPUT of iptables too. default-DROP would block IPIP/VXLAN/WG/etc. traffic
-		// without explicit ACCEPT.
-		inputRules = append(inputRules, iptables.Rule{
-			Match:  iptables.Match().MarkMatchesWithMask(tcdefs.MarkSeen, tcdefs.MarkSeenMask),
-			Action: iptables.AcceptAction{},
-		})
+		if rulesConfig.EndpointToHostAction != "ACCEPT" {
```

Comment:
What about traffic arriving on eth0, i.e. traffic to the host but not from a workload?  Will that get caught by this rule change?

Reply:
In that case, if INPUT is DROP it will get dropped unless it is wireguard. afaict it is what would happen in iptables. It is covered by the wg tests. If default hepToHost is ACCEPT, it will get accepted by other rules sooner.